swift/ql/lib/codeql/swift/generated/expr/OverloadedDeclRefExpr.qll

```diff
@@ -1,10 +1,45 @@
 // generated by codegen/codegen.py
 private import codeql.swift.generated.Synth
 private import codeql.swift.generated.Raw
-import codeql.swift.elements.expr.OverloadSetRefExpr
+import codeql.swift.elements.expr.Expr
+import codeql.swift.elements.decl.ValueDecl
 
 module Generated {
-  class OverloadedDeclRefExpr extends Synth::TOverloadedDeclRefExpr, OverloadSetRefExpr {
+  /**
+   * An ambiguous expression that might refer to multiple declarations. This will be present only
+   * for failing compilations.
```

Comment:
So will it ever actually be in a database?

Reply:
I think not, not unless:
* there's an actual failure in the compilation, or
* there's a bug in our extractor

Reply:
I've opened [another PR](https://github.com/github/codeql/pull/11196) on top of this one to organize all elements that can only come from some kind of error in a common `ErrorElement` superclass.